ci/release/update-version.sh

```diff
@@ -75,3 +75,10 @@ for FILE in .github/workflows/*.yaml; do
 done
 
 sed_runner "s/--rapids-version=[[:digit:]]\{2\}.[[:digit:]]\{2\}/--rapids-version=${NEXT_RAPIDS_SHORT_TAG}/g" .pre-commit-config.yaml
+
+# .devcontainer files
+find .devcontainer/ -type f -name devcontainer.json -print0 | while IFS= read -r -d '' filename; do
+    sed_runner "s@rapidsai/devcontainers:[0-9.]*@rapidsai/devcontainers:${NEXT_RAPIDS_SHORT_TAG}@g" "${filename}"
+    sed_runner "s@rapidsai/devcontainers/features/rapids-build-utils:[0-9.]*@rapidsai/devcontainers/features/rapids-build-utils:${NEXT_RAPIDS_SHORT_TAG_PEP4400}@" "${filename}"
```

Comment:
```suggestion
    sed_runner "s@rapidsai/devcontainers/features/rapids-build-utils:[0-9.]*@rapidsai/devcontainers/features/rapids-build-utils:${NEXT_RAPIDS_SHORT_TAG_PEP440}@" "${filename}"
```

Reply:
blegh, this mistake shows that the way I tested in https://github.com/rapidsai/ucxx/pull/363#discussion_r1939570571 wasn't great. There I was checking for "nothing was missed", should have also manually scanned the diff for "all the updates look right".

Fixed in https://github.com/rapidsai/ucxx/pull/363/commits/3c774c2314d6fee7d3b68440af17b838b439d989

Sorry for making you catch these things in review, I know it's annoying :/

Reply:
No worries! Thanks for your work on this PR. :)